service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -275,6 +266,19 @@ public ResponseEntity<ApiWorkspaceDescription> getWorkspaceByUserFacingId(
     return new ResponseEntity<>(desc, HttpStatus.OK);
   }
 
+  @Override
+  public ResponseEntity<Void> updateWorkspaceProperties(
```

Comment:
can you add a test in workspaceapicontrollertest thanks

Reply:
1 is fine thx

it's not enforced by this repo but generally i like all the public method to be tested.